Update developing-service-configuration-apps.md (#608)
vtex-apps/io-documentation: docs/en/Recipes/development/developing-service-configuration-apps.md

4. Open the app using your code editor;
5. In the `node/service.json` file, add `"settingsType": "workspace"` to the app's path to define which routes will be able to receive configurations through requests. You should end up with something similar to the example below:

```diff
 {
   "routes": {
     "status": {
       "path": "/_v/status/:code",
       "public": true,
+      "settingsType": "workspace"
     }
   }
 }
```

It is also possible to **define your configurations through event listening**. For this scenario, you should add in the  `node/service.json` file something similar to the example below, replacing the values according to your needs: 

```diff
 {
   "events": {
     "eventHandler": {
       "sender": "appEmittingTheEvent",
       "keys": ["topic"],
+      "settingsType": "workspace"
     }
   }
 }
```

6. In the `manifest.json` file, add the `configuration` Builder to the `builders` list and update the app's name to one of your choosing. For example:

``` diff
 {
   "name": "vtex.most-amazing-service-ever",
   "version": "0.0.0",
   "builders": {
     "node": "4.x",
+    "configuration": "0.x"
   }
 }
```

> 7. Create a `schema.json` file in the `configuration` folder. You will need this `configuration/schema.json` file to define the settings structure that the service app is going to accept from other apps on the platform.

Author: it's the folder for the builder, in the root of the app. yes, it's needed to create the folder before creating the schema file

> 7. Create a `schema.json` file in the `configuration` folder. You will need this `configuration/schema.json` file to define the settings structure that the service app is going to accept from other apps on the platform.

Reviewer: ```suggestion
7. Create a `configuration` folder in your app's root directory, and, then, a `schema.json` file inside it. This file will hold information about the settings structure that the service app is going to accept from other apps on the platform.
```

> 8. Once the file is created, create a JSON Schema in that file, according to your scenario. For example:

Reviewer: ```suggestion
8.  In the `configuration/schema.json` file, create a JSON Schema, according to your scenario. For example:
```

> 1. In your VTEX IO app, add the recently created service app as a new builder. For example:

Reviewer: ```suggestion
1. In your VTEX IO configuration app, add the recently created service app as a new builder. For example:
```

```diff
 {

>    "name": "vtex.amazing-service-configuration",

Reviewer 2: _nit_: maybe only `vtex.amazing-configuration` can be clearer

Author: done! :-)
